problems/3.conditionals.js

```diff
@@ -2,25 +2,31 @@
 * @param {number} temp - The current temperature
 * @returns {string} - A message about the temperature
 */
-function temperatureMessage() {
-
+function temperatureMessage(temp) {
+  if (temp < 32) {
+    return "It's freezing!"
+  } else return "It's alright"
```

Comment:
Use curly braces on else. It's easier and prettier to read (unless the if was also a one liner)